packages/db/fauna/resources/Function/createBatch.js

```diff
@@ -0,0 +1,81 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Let,
+  Select,
+  Var,
+  If,
+  Create,
+  Update,
+  Exists,
+  Collection,
+  Foreach,
+  Index,
+  IsEmpty,
+  Abort,
+  Match,
+  Do,
+  Get
+} = fauna
+
+const name = 'createBatch'
+const body = Query(
+  Lambda(
+    ['data'],
+    Let(
+      {
+        batchCid: Select('batchCid', Var('data')),
+        pieceCid: Select('pieceCid', Var('data')),
```

Comment:
When I call this API I know the exact byte-size of the resulting car (I can't calculate the PieceCID without the file - so I obviously know its size). You could store that info if of interest.